src/client/templates/MainTemplate.jsx

```diff
@@ -16,7 +16,8 @@ const MainTemplate = ({ route }) => (
       brand={
         <Link to="/">
           <h1 className="sr-only">Grace Covenant Church</h1>
-          <img id="logo" src="/static/images/gcclogo.png" alt="Grace Covenant Church" />
+          <img className={`${navStyles['logo']}`} src="/static/images/gcclogo.png" alt="Grace Covenant Church" />
```

Comment:
nit: I think it will be better to name these styles `logo-light` and `logo-dark`. `logo` and `logo-black` make it seem like `logo-black` specializes the styling of `logo` whereas they're essentially disjoint styles.